src/pages/graphql/schema/cart/mutations/place-order.md

```diff
@@ -47,8 +47,12 @@ mutation {
       cart_id: "IeTUiU0oCXjm0uRqGCOuhQ2AuQatogjG"
     }
   ) {
-    order {
-      order_number
+    orderV2 {
```

Comment:
Add a paragraph immediately above the InlineArt in the description section stating that as of 2.4.7, the PlaceOrderOutput object can contain orderV2, which contains full details about the order.

Reply:
Done!